src/services/OAUTH2.ts

```diff
@@ -8,7 +8,7 @@ export const redirectToAutorizationPage = (
 ) => {
   window.location.href = `https://www.reddit.com/api/v1/authorize?client_id=${clientId}&response_type=code&state=${generateRandomState(
     16,
-  )}&redirect_uri=${redirectURI}&scope=identity&duration=temporary`
+  )}&redirect_uri=${redirectURI}&scope=identity%20account&duration=temporary`
```

Comment:
I think all parameters shouldn't be hardcoded

Reply:
fixed